src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -0,0 +1,227 @@
+      boost::property_tree::ptree& graph = gr.second;
+      _output << boost::format("  GRAPH[%2d] Name:%2s\n") % graph.get<std::string>("id")
+           % graph.get<std::string>("name");
+      _output << boost::format("    SNo.  %-20s%-30s%-30s\n") % "Core [C:R]"
+           % "Iteration_Memory [C:R]" % "Iteration_Memory_Addresses";
+      int count = 0;
+      for (auto& node : graph.get_child("tile")) {
+        boost::property_tree::ptree& tile = node.second;
+        _output << boost::format("    [%2d]   %-20s%-30s%-30d\n") % count
+            % (tile.get<std::string>("col")+":"+tile.get<std::string>("row"))
+            % (tile.get<std::string>("memcol")+":"+tile.get<std::string>("memrow"))
+            % node.second.get<uint16_t>("memaddr");
+        count++;
+      }
+      _output << std::endl;
+
+      _output << boost::format("    %s\n") % "Pl Kernel Instances in Graph:";
+      for (auto& node : graph.get_child("pl_kernel")) {
+        _output << boost::format("      %s\n") % node.second.data();
+      }
+      _output << std::endl;
+    }
+
+    int count = 0;
+    for (auto& rtp_node : _pt.get_child("aie_metadata.rtps")) {
+      _output << boost::format("  %-3s:[%2d]\n") % "RTP" % count;
+      _output << boost::format("    %-22s: %s\n") % "Port Name" % rtp_node.second.get<std::string>("port_name");
+      _output << boost::format("    %-22s: %d\n") % "Selector Row" % rtp_node.second.get<uint16_t>("selector_row");
+      _output << boost::format("    %-22s: %d\n") % "Selector Col" % rtp_node.second.get<uint16_t>("selector_column");
+      _output << boost::format("    %-22s: %d\n") % "Selector Lock Id" % rtp_node.second.get<uint16_t>("selector_lock_id");
+      _output << boost::format("    %-22s: %d\n") % "Selector Addr" %  rtp_node.second.get<uint64_t>("selector_address");
+      _output << boost::format("    %-22s: %d\n") % "Ping Buffer Row" % rtp_node.second.get<uint16_t>("ping_buffer_row");
+      _output << boost::format("    %-22s: %d\n") % "Ping Buffer Col" % rtp_node.second.get<uint16_t>("ping_buffer_column");
+      _output << boost::format("    %-22s: %d\n") % "Ping Buffer Lock Id" % rtp_node.second.get<uint16_t>("ping_buffer_lock_id");
+      _output << boost::format("    %-22s: %d\n") % "Ping Buffer Addr" % rtp_node.second.get<uint64_t>("ping_buffer_address");
+      _output << boost::format("    %-22s: %d\n") % "Pong Buffer Row" % rtp_node.second.get<uint16_t>("pong_buffer_row");
+      _output << boost::format("    %-22s: %d\n") % "Pong Buffer Col" % rtp_node.second.get<uint16_t>("pong_buffer_column");
+      _output << boost::format("    %-22s: %d\n") % "Pong Buffer Lock Id" % rtp_node.second.get<uint16_t>("pong_buffer_lock_id");
+      _output << boost::format("    %-22s: %d\n") % "Pong Buffer Addr" % rtp_node.second.get<uint64_t>("pong_buffer_address");
+      _output << boost::format("    %-22s: %b\n") % "Is Plrtp" % rtp_node.second.get<bool>("is_PL_RTP");

```

Comment:
Can what "Plrtp" be more descriptive on what it represents?

Reply:
done